_targets_eval_postprocessing.R

```diff
@@ -1116,8 +1109,11 @@ hub_targets <- list(
         ),
         to = lubridate::ymd(max(eval_config$forecast_date_hosp)),
         by = "week"
-      )
-    )
+      ) # Ensure that local retrospective hub submission files have been made
+    ) |> with_dependencies(c(
+      metadata_hub_submissions,
+      metadata_hosp_hub_submissions
+    ))
```

Comment:
Don't need to concatenate them since `with_dependencies` takes arbitrarily many additional arguments
```suggestion
    ) |> with_dependencies(
      metadata_hub_submissions,
      metadata_hosp_hub_submissions)
```